addons/pos_discount/models/pos_config.py

```diff
@@ -7,19 +7,11 @@
 class PosConfig(models.Model):
     _inherit = 'pos.config'
 
+    def _default_discount_product_id(self):
+        return self.env.ref("point_of_sale.product_product_consumable")
```

Comment:
it will traceback if the "product_product_consumable" you should probably do something like return self.env.ref('point_of_sale.product_product_consumable', raise_if_not_found=False) to return Flase if the product is does not exist.